packages/orbit-components/tailwind.config.ts

```diff
@@ -6,7 +6,7 @@ export default {
   content: ["./src/**/*.{ts,tsx}"],
   presets: [
     orbitPreset({
-      disablePreflight: false,
+      disablePreflight: process.env.NODE_ENV === "test",
```

Comment:
Won't this always be `true` for us? Does it mean we change the behavior we had?

Reply:
When running storybook, the NODE_ENV is not `test` so this is false, and therefore preflight is enabled.
When we execute `yarn components tailwind`, that should be executed before testing, the NODE_ENV is set to `test` and so preflight will be disabled